app/templates/package.json

```diff
@@ -23,27 +23,29 @@
   },
   "homepage": "https://github.com/<%= user %>/<%= repo %>",
   "devDependencies": {
-    "babel-core": "^5.2.17",
-    "babel-eslint": "^4.0.5",
-    "babel-loader": "^5.3.2",
-    "chai": "^3.2.0",
+    "babel-core": "^6.3.26",
```

Comment:
Is babel-core still a dep? If it is may as well continue using `babel-core/register` imo